tests/baselines/reference/findAllReferencesLinkTag1.baseline.jsonc

```diff
@@ -0,0 +1,857 @@
+//      * {@link C.prototype.m}
+//      * @see {C.prototype.m}
+//      */
+//     p() { }
+//     /**
+//      * {@link n}
+//      * @see {n}
+//      * {@link C.[|n|]}
+//      * @see {C.[|n|]}
+//      * {@link C#n}
+//      * @see {C#n}
+//      * {@link C.prototype.n}
+//      * @see {C.prototype.n}
+//      */
+//     q() { }
+//     /**
+//      * {@link s}
+//      * @see {s}
+//      * {@link C.s}
+//      * @see {C.s}
+//      */
+//     r() { }
+// }
+// 
+// interface I {
+//     a()
+//     b: 1
+//     /** {@link a} @see {a} */
+//     c()
+//     /** {@link b} @see {b} */
+//     d()
+// }
+// 
+// function nestor() {
+//     /** {@link r2} */
+//     function ref() { }
+//     /** @see {r2} */
+//     function d3() { }
+//     function r2() { }
+// }
+
+[
+  {
+    "definition": {
+      "containerKind": "",
+      "containerName": "",
+      "fileName": "/tests/cases/fourslash/findAllReferencesLinkTag1.ts",
+      "kind": "property",
+      "name": "(property) C.n: number",
+      "textSpan": {
+        "start": 26,
+        "length": 1
+      },
+      "displayParts": [
+        {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "property",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "C",
+          "kind": "className"
+        },
+        {
+          "text": ".",
+          "kind": "punctuation"
+        },
+        {
+          "text": "n",
+          "kind": "propertyName"
+        },
+        {
+          "text": ":",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "number",
+          "kind": "keyword"
+        }
+      ],
+      "contextSpan": {
+        "start": 26,
+        "length": 5
+      }
+    },
+    "references": [
+      {
+        "textSpan": {
+          "start": 26,
+          "length": 1
+        },
+        "fileName": "/tests/cases/fourslash/findAllReferencesLinkTag1.ts",
+        "contextSpan": {
+          "start": 26,
+          "length": 5
+        },
+        "isWriteAccess": true,
+        "isDefinition": true
+      },
+      {
+        "textSpan": {
+          "start": 300,
+          "length": 1
+        },
+        "fileName": "/tests/cases/fourslash/findAllReferencesLinkTag1.ts",
+        "isWriteAccess": false,
+        "isDefinition": false
+      },
+      {
+        "textSpan": {
+          "start": 318,
+          "length": 1
+        },
+        "fileName": "/tests/cases/fourslash/findAllReferencesLinkTag1.ts",
+        "isWriteAccess": false,
+        "isDefinition": false
+      }
+    ]
+  }
+]
+
+// === /tests/cases/fourslash/findAllReferencesLinkTag1.ts ===
+// class C {
+//     m() { }
+//     n = 1
+//     static [|s|]/*FIND ALL REFS*/() { }
```

Comment:
So this is not supposed to light up the `C.s` references?

Reply:
But it does! (In my editor!!) (Similar problem as above I suspect.)

Reply:
This one isn’t a baseline writing bug—no references (besides the one where it was triggered) is represented in the JSON.

Reply:
As far as I can tell, this test should be treated as failing.

Reply:
Well, "it works on my machine" turns out to be "actually broken" as usual. Latest commit adds code to make it work and updates the baseline.